doc/mlsys_paper/paper.tex

```diff
@@ -398,24 +408,26 @@ \section{Example: Learning Linear Regression Models}
     };
 \end{minted}
 \vspace*{-0.4em}
+%% CS: the above code is currently shown across two pages - not a clean look.
+%% CS: how about we place it in a figure ?
```

Comment:
A figure's fine with me.  I liked it inline if it was on one page since it keeps the linear flow of reading, but if it splits over a page, you're right, it's a little awkward.  (Also, the other code is inline and probably should stay that way.  Instead do you think we should aim to just get all the code for `LinearRegressionFunction` on one page and leave it inline?)

Reply:
I'd strongly prefer to have large code listings (ie. more than, say, 4 or 5 lines) to be figures. It looks more professional and cleaner.  It's also makes it super clear what we're referring to by using "the code shown in Fig. X", rather than relying on the reader to accept that the "funny looking" stuff below is actually a continuation of the preceding text.

There's an effect known as [Banner Blindness](https://en.wikipedia.org/wiki/Banner_blindness), where people skip over stuff that seems out of place.  This is exactly what I'm experiencing when seeing this big code chunk -- my eyes want to skip over it and get back to reading the "real" text.  If I'm experiencing this, there's a good chance a reviewer may as well.

Reply:
I'll handle this one tomorrow morning unless you want to handle it while I'm asleep. :)

Reply:
I'll do a bunch of PRs, probably spaced over 10 hours or so.

Reply:
Sounds good. I am headed to bed now, so I definitely won't be able to reapproach this within 10 hours. One PR is fine too, up to you.

Reply:
1 large-ish PR done. sent email with more info and suggestions for improving the flow